src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -3497,6 +3452,14 @@ MavlinkReceiver::updateParams()
 	if (_handle_sens_flow_rot != PARAM_INVALID) {
 		param_get(_handle_sens_flow_rot, &_param_sens_flow_rot);
 	}
+
+	if (_handle_ekf2_min_rng != PARAM_INVALID) {
```

Comment:
Let's add `SENS_RNG_MIN` or something, I don't think we should let ekf2 parameters leak in like this.

Reply:
But then the params are not in sync and that might cause problems.

Reply:
They're different things that happen to be the same in the mantis case (I believe). EKF2_MIN_RNG is the range finder ground clearance, it's the value you expect to have when landed. Separately there's a minimum distance that a sensor is capable of reading, and anything below that is considered invalid.

Aside from that we shouldn't make mavlink depend on ekf2.

If you don't want to get into adding the new parameters (they'll probably need to be generalized for multiple instances soon) I would just set the hard coded min_distance to 0.01 for now.

Reply:
Ok, I can do that. And what about the max_distance?